Libs/MRML/DisplayableManager/vtkMRMLModelDisplayableManager.cxx

```diff
@@ -417,6 +421,12 @@ int vtkMRMLModelDisplayableManager::UpdateClipSlicesFromMRML()
     this->Internal->YellowSliceClipState = this->Internal->ClipModelsNode->GetYellowSliceClipState();
     }
 
+  if (this->Internal->ClipModelsNode->GetClippingMethod() != this->Internal->ClippingMethod)
+    {
+    modifiedState = 1;
```

Comment:
Why do you need this additional state variable? All modification logic has to be managed by modification timestamps.